pkg/operator/webhookconfig.go

```diff
@@ -78,57 +78,59 @@ func (w *WebhookConfigUpdater) Start(stopCh <-chan struct{}) error {
 }
 
 // ValidatingWebhooks returns the validating webhook configurations.
-func (w *WebhookConfigUpdater) ValidatingWebhooks() ([]admissionregistrationv1beta1.ValidatingWebhook, error) {
-	failurePolicy := admissionregistrationv1beta1.Ignore
-	sideEffects := admissionregistrationv1beta1.SideEffectClassNone
+func (w *WebhookConfigUpdater) ValidatingWebhooks() ([]admissionregistrationv1.ValidatingWebhook, error) {
+	failurePolicy := admissionregistrationv1.Ignore
+	sideEffects := admissionregistrationv1.SideEffectClassNone
 
-	webhooks := []admissionregistrationv1beta1.ValidatingWebhook{
+	webhooks := []admissionregistrationv1.ValidatingWebhook{
 		{
-			Name: "clusterautoscalers.autoscaling.openshift.io",
-			ClientConfig: admissionregistrationv1beta1.WebhookClientConfig{
-				Service: &admissionregistrationv1beta1.ServiceReference{
+			Name:                    "clusterautoscalers.autoscaling.openshift.io",
+			AdmissionReviewVersions: []string{"v1", "v1beta1"},
+			ClientConfig: admissionregistrationv1.WebhookClientConfig{
+				Service: &admissionregistrationv1.ServiceReference{
 					Name:      fmt.Sprintf("%s-operator", OperatorName),
 					Namespace: w.namespace,
 					Path:      pointer.StringPtr("/validate-clusterautoscalers"),
 				},
 			},
 			FailurePolicy: &failurePolicy,
 			SideEffects:   &sideEffects,
-			Rules: []admissionregistrationv1beta1.RuleWithOperations{
+			Rules: []admissionregistrationv1.RuleWithOperations{
 				{
-					Rule: admissionregistrationv1beta1.Rule{
+					Rule: admissionregistrationv1.Rule{
 						APIGroups:   []string{"autoscaling.openshift.io"},
 						APIVersions: []string{"v1"},
 						Resources:   []string{"clusterautoscalers"},
 					},
-					Operations: []admissionregistrationv1beta1.OperationType{
-						admissionregistrationv1beta1.Create,
-						admissionregistrationv1beta1.Update,
+					Operations: []admissionregistrationv1.OperationType{
+						admissionregistrationv1.Create,
+						admissionregistrationv1.Update,
 					},
 				},
 			},
 		},
 		{
-			Name: "machineautoscalers.autoscaling.openshift.io",
-			ClientConfig: admissionregistrationv1beta1.WebhookClientConfig{
-				Service: &admissionregistrationv1beta1.ServiceReference{
+			Name:                    "machineautoscalers.autoscaling.openshift.io",
+			AdmissionReviewVersions: []string{"v1", "v1beta1"},
```

Comment:
IIUC we want to just support v1 going forward as v1beta1 will be removed in 1.22
```suggestion
			AdmissionReviewVersions: []string{"v1"},
```